code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -157,7 +157,9 @@
 	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
 	item_flags = NEEDS_PERMIT | ABSTRACT | NODROP | DROPDEL
 	w_class = WEIGHT_CLASS_HUGE
+	total_mass = 3.5
```

Comment:
_3.5????_ For a sharp weapon that crits in 4 hits and is only available to an antag type that has very easy access to spammable adrenals? Adrenals that regen a total of 30(!!!) stamloss per mob life cycle (even in combat mode!) and come with a burst of 150 stamina regen?
Who are you trying to balance for?

Reply:
It's got 25 force, but at the other hand, it used to have zero armor penetration and it still offers absolutely nothing in terms of defensive utility, unlike an e-sword, for example. It's a loud as fuck weapon which also makes you supervalid to silicons too.
Though, after learning how armor-pen works, imma lower the value a bit.